camel/agents/tool_agents/hugging_face_tool_agent.py

```diff
@@ -171,7 +175,7 @@ def step(
         if remote is None:
             remote = self.remote
         agent_output = self.agent.run(*args, remote=remote, **kwargs)
-        if isinstance(agent_output, AgentImage):
+        if isinstance(agent_output, self.AgentImage):
```

Comment:
Pls revert. What is the purpose of saving a type in a field?

Reply:
> Move all HF related imports here.

I want all hugging face imports inside the class init. I must assign it to a field in order to use it in the later code.